### ISSUE TITLE
remove Arabic words from Persian language!!!

### ISSUE THREAD

**Issue by Morteza-NW** [aradzie/keybr.com#317]
`as one of the Iranian users who use this amazing site every day, why are there Arabic words inside the Persian text? I know we have many similarities, but we don’t use the word "al" before our words. This is one of the problems that bothers me a lot.

**Comment by aradzie** [comment]
Well I don't speak neither languages, I need help from a native speaker. Can you please review the word list and update it?

**Comment by Morteza-NW** [comment]
Yeah sure, why not? I'm a native Persian speaker, and it would be great to help my people on this amazing typing site. How can I review words?

**Comment by Morteza-NW** [comment]
> Well I don't speak neither languages, I need help from a native speaker. Can you please review the word list and update it?

even now I'll gladly accept and help you with the words, its quite hard to improve your Persian touch typing with Arabic,
call me whenever and wherever you want to change it.

**Comment by aradzie** [comment]
@Morteza-NW

**Comment by Morteza-NW** [comment]
Hi [Aleksander],

Thank you for your message and for providing the details regarding the Persian word list changes.

I understand that the deleted words are similar to English words with "ing" and "ed." While they are not Arabic, they are forms of Persian words used in sentences. I agree that removing these words is appropriate for typing practice, as they do not contribute effectively to the learning process.

I am happy to help review the words. Please note that it will take some time as I will be working on this during my free time, but I assure you that I will deliver a complete and correct form.

I appreciate your patience and understanding.

Best regards,

Morteza

**Comment by Morteza-NW** [comment]
Hi Aleksander,
@aradzie 

Thank you for your message and the updates to the Persian word list.

I have thoroughly reviewed and corrected the Google Sheet, and I removed around 300 words that were offensive or not relevant. This should now provide a better experience for Persian users.

Additionally, I have a suggestion: It might be beneficial to replace the hyphen (-) with the Persian version known as "ویرگول" (Virgol). We commonly use it for separating items, much like a comma (,) in English.

We also frequently use "تشدید" (Tashdid) and "آ" (A) in Persian. If you want to include these, you should add separate words that contain them into the sheet.

Please let me know if there’s anything else you need.

Best regards, Morteza😁

**Comment by aradzie** [comment]
I have updated the website with a new Persian word list. Thanks!
